packages/e2e-tests/fixtures/block-transforms.js

```diff
@@ -0,0 +1,178 @@
+export const EXPECTED_TRANSFORMS = {
+	core__archives: {
```

Comment:
It's too bad we need to keep a list of possible transforms. Each change will have to be recorded here as well. Why not do:

* Create a block.
* See if there are transforms (and count the items in the list).
* Transform by clicking the first item, then transform back.
* Continue until all transforms are tried.
* Do the same for all registered blocks.

Reply:
Hi @ellatrix thank you for thinking and sharing your thoughts on this.
There are some reasons for this approach:
- If a bug stops showing a transform in the UI our tests would not catch it.
- We can not generate test cases based on an async condition, e.g: waiting for for the page to render and query the transforms available. So a test would look like "transforms work as expected". When it fails the information provided to the user would not clearly specify which transforms failed. Having this file we can generate a test per transform that looks like "Quote block should transform to List block. fixture: core__quote__style-1". Which clearly identifies the transform that failed.
- Some of our transforms are not symmetric, transform block A to B and transforming B back to A  does create a block equal to the starting point. E.g: Quote -> Heading, Heading -> Quote. So we would still need snapshots with the results of the transform.

We need to update this file when a new fixture is added, and when a transform is added. This are not very common operations.